votor-messages/src/migration.rs

```diff
@@ -253,6 +253,12 @@ impl MigrationPhase {
         self.is_alpenglow_block(slot)
     }
 
+    /// Should this block allow the UpdateParent marker, i.e., support fast leader handover?
+    fn should_allow_fast_leader_handover(&self, slot: Slot) -> bool {
+        // Only allow for alpenglow blocks, 2 leader windows after after migration
+        self.is_alpenglow_block(slot.saturating_sub(2 * NUM_CONSECUTIVE_LEADER_SLOTS))
```

Comment:
why can't we enable it immediately, what breaks? 
```suggestion
        self.is_alpenglow_block(slot)
```